third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -571,11 +571,21 @@ pub enum StatusCode {
     MULTISIG_TRANSACTION_INSUFFICIENT_APPROVALS = 34,
     MULTISIG_TRANSACTION_PAYLOAD_DOES_NOT_MATCH_HASH = 35,
     GAS_PAYER_ACCOUNT_MISSING = 36,
+    // ZkID account specific errors
+    EPHEMERAL_KEYPAIR_EXPIRED = 37,
+    INVALID_JWT_HEADER = 38,
+    JWT_VERIFICATION_FAILED = 39,
+    TOO_MANY_ZK_AUTHENTICATORS = 40,
+    FAILED_TO_GET_ON_CHAIN_CURRENT_TIME = 41,
+    FAILED_TO_GET_ON_CHAIN_JWKS = 42,
+    FAILED_TO_FIND_JWK = 43,
+    UNSUPPORTED_JWK = 44,
```

Comment:
are we supposed to touch it? it seems the pure move area that shouldn't be aware of this feature

Reply:
Yep, wondering the same thing...